### DISCUSSION TITLE
dns record dont found yn domains

### DISCUSSION THREAD

**Discussion by operacionesBlueCorner** [vercel/community#6333] [Help]
### Summary

Login to my vercel account.
I choose the project, but dns records do not appear, to add some changes, yesterday it appeared normal, today it stopped appearing.
I have 2 projects and it does not appear in either of them.

![image](https://github.com/vercel/community/assets/96430730/1473540c-4538-4807-88d5-01a59fc51f87)

### Steps to Reproduce

Domains in Go Daddy

**Comment by amyegan** [comment]
It sounds like your account may have been updated to our new structure today. If that's the case, your domain might need to be moved from your personal account to your new hobby team. You can learn more about this account structure change in https://github.com/vercel/community/discussions/6298
